app/page.tsx

```diff
@@ -1,113 +1,133 @@
-import Image from 'next/image'
+import Image from "next/image";
```

Comment:
I'm curious why you chose to switch these to double quotes. Both are correct. I often do this too... my OCD tendencies can make it feel REALLY important. 

It's a good example of a team deciding on code styles. Many styles might work, but it's nice to agree on some things so the code looks uniform... like:

* How many spaces is a tab? 
* Single or double quotes (double are often more flexible, allowing single quotes inside of them)?

Often, VSCode will have defaults for different types of code (HTML, JS/TS, PHP, etc.) and that's a good starting point. We can worry about this explicitly later if we start seeing clashing standards that distract. For now, just good to have in mind.

Reply:
I don't even remember switching this to double quotes, it might just be how my 'prettier' extension decides to auto format it.